.github/workflows/dart.yml

```diff
@@ -13,7 +13,9 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        sdk: [stable, 2.7.2, 2.12.0-133.7.beta, beta, dev ]
+        # Test latest stable, beta, and dev channels + last stable release
+        # prior to the introduction of the unified `dart` developer tool.
+        sdk: [stable, beta, dev, 2.9.3]
```

Comment:
Technically, removing the specific beta means we're no longer testing extracting the channel from the version.

Reply:
Good point, added back (used the first null safety beta release)